### PR TITLE
Nits: fix clippy warnings

### PR DESCRIPTION
**Pull request by xortive**
Cleaned up some clippy warnings that got past CI.

### CODE REVIEW

File: src/terminal/message.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```
Comment (xortive) [inline src/terminal/message.rs]: normally would be against this type of hammer, but I think this is an appropriate nail.

Comment (EverlastingBugstopper) [reply src/terminal/message.rs]: i'm confused is it a hammer or a nail 😉

File: src/commands/publish/package.rs [hunk 2]
```diff
@@ -38,7 +38,7 @@ impl Package {
 
         let package_json: String = fs::read_to_string(manifest_path.clone())?.parse()?;
         let package: Package = serde_json::from_str(&package_json)
-            .expect(&format!("could not parse {:?}", manifest_path));
+            .unwrap_or_else(|_| panic!("could not parse {:?}", manifest_path));
```
Comment (gabbifish) [inline src/commands/publish/package.rs]: Out of curiosity, why did `expect` cause these warnings?

Comment (xortive) [reply src/commands/publish/package.rs]: https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call

**Pull request merged**